app/src/main/java/com/spotlight/spotlightapp/splash/SplashScreenFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.spotlight.spotlightapp.splash
+
+import android.os.Bundle
+import android.view.View
+import androidx.compose.ui.platform.ComposeView
+import androidx.core.os.bundleOf
+import androidx.fragment.app.Fragment
+import com.spotlight.spotlightapp.R
+
+class SplashScreenFragment : Fragment(R.layout.fragment_splash_screen) {
+    companion object {
+        const val REQUEST_KEY = "SplashScreenFragmentRequest"
+        const val IS_FINISHED_SPLASH_ANIMATION = "is_finished_splash_animation"
+    }
+
+    private val splashScreenFragmentView: SplashScreenFragmentView by lazy {
+        SplashScreenFragmentView()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<ComposeView>(R.id.composeView).setContent {
```

Comment:
Use kotlin synthetic for now while viewBinding is still not yet available as a plugin (due to compose incompatibilities)

Reply:
With the current Kotlin version, Kotlin synthetics does not go well with Jetpack Compose https://stackoverflow.com/questions/63788205/migrate-to-jetpack-compose-error-unbound-symbols-not-allowed